products.js

```diff
@@ -16,21 +16,27 @@
 
 const {readFileSync, writeFile, readdirSync, statSync} = require('fs');
```

Comment:
I'm generally confused by this commit. The commit message says this commit is all about splitting the corpus, but there appear to be a lot of changes here that have nothing to do with that. In the future, could you could break up your commits or at least outline all the changes in the commit message?